app/Http/Controllers/ArticleController.php

```diff
@@ -73,10 +74,12 @@ public function show(Article $article)
      * @param  \App\Models\Article  $article
      * @return \Illuminate\Http\RedirectResponse
      */
-    public function edit(Article $article)
+    public function edit($id)
```

Comment:
```suggestion
    public function edit(int $id)
```

上のphpdocも修正必要。PHPStorm使って自動生成したい